src/index.html

```diff
@@ -11,6 +11,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      class="search-form">
+      <img
+        src="images/Search.svg"
+        alt="search-icon"
+        class="search-icon"
+      >
+      <input
+        type="text"
+        class="input"
+        placeholder= "Try &#34;Los Angeles&#34;"
+        data-qa="big"
+      >
+    </form>
+
+    <form
+      action="#"
+      method="get"
+      class="search-form">
+      <img
```

Comment:
[Checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

```suggestion
    <form
      action="#"
      method="get"
      class="search-form"
    >
      <img
```